server/src/replay/recording/ActionRecord.ts

```
@@ -0,0 +1,276 @@
+        databaseGameActionRecord.statistics.singleplayer = {
+          score: data.score,
+          timeInMilliseconds: data.elapsedTime,
+          scoreSubmissionDateAndTime: timestamp,
+          enemiesCreated: data.enemiesSpawned,
+          enemiesKilled: data.enemiesKilled,
+          actionsPerformed: data.actionsPerformed
+        };
+        break;
+      }
+      case "defaultMultiplayer": {
+        if (!(data instanceof Array)) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)